plugins/net_plugin/net_plugin.cpp

```diff
@@ -2664,6 +2698,8 @@ namespace eosio {
          boost::asio::bind_executor( strand,
                [resolver, c = shared_from_this(), socket=socket]( const boost::system::error_code& err, const tcp::endpoint& endpoint ) {
             if( !err && socket->is_open() && socket == c->socket ) {
+               c->update_endpoints();
+               c->connection_time = get_time();
```

Comment:
`connection_time` does not appear to be set for incoming connections.

Reply:
Updated.